resources/js/alpine/linkifyDirective/linkifyDirective.ts

```diff
@@ -0,0 +1,38 @@
+import linkifyHtml from 'linkify-html';
+import type { DirectiveData, DirectiveUtilities } from 'alpinejs';
+
+export function linkifyDirective(
+  el: HTMLElement,
+  _: DirectiveData,
+  { effect }: DirectiveUtilities,
+) {
+  effect(() => {
+    el.innerHTML = linkifyHtml(el.innerHTML, {
+      formatHref: {
+        url: (href) => {
+          if (href.toLowerCase().includes('retroachievements.org')) {
+            return href;
+          }
+
+          return `/redirect?url=${href}`;
+        },
+      },
+
+      validate: {
+        url: (text) => {
+          const sanitized = text.toLowerCase();
+
+          if (sanitized.endsWith('.mp') || sanitized.endsWith('.ar')) {
+            return false;
+          }
```

Comment:
While this does address specific cases I found, it feel like a kludge. I'd rather have a few unexpected links then have to maintain this.

Reply:
I've removed this `verify` logic in latest - all TLDs will now be linkified.